pages/common/file-rename.md

```diff
@@ -0,0 +1,7 @@
+# file-rename
+
+> This command is an alias of `rename`.
+
+- View documentation for the original command:
+
+`tldr rename.file`
```

Comment:
```suggestion
`tldr -p common rename`
```
`rename.file.md` doesn't exist any more so we'll have to cut the end
Also lets put the platform instructions there too. Someone might end on this page from linux, then get redirected to the disambiguation page and then get redirected to the correct page. Lets cut the middleman.